src/HomePage.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
nit: if these items are not along the same line in the view, can we get vertical spacing between them?

```
<h1>Yo</h1>

<div>Yo</div>
```
instead of 
```
<h1>Yo</h1>
<div>Yo</div>
```